pkgs/applications/audio/roomeqwizard/default.nix

```diff
@@ -0,0 +1,91 @@
+{ writeTextFile, jdk8, runCommand, fetchurl, lib, makeWrapper, writers
+, common-updater-scripts, perl, nixfmt, nix, curl, coreutils, makeDesktopItem }:
+
+let
+  pname = "roomeqwizard";
+  version = "5.20.4";
+
+  responseFile = writeTextFile {
+    name = "response.varfile";
+    text = ''
+      createDesktopLinkAction$Boolean=false
+      executeLauncherAction$Boolean=false
+      mem$Integer=1
+      opengl$Boolean=false
+      sys.adminRights$Boolean=false
+      sys.installationDir=INSTALLDIR
+      sys.languageId=en
+      sys.programGroupDisabled$Boolean=true
+        '';
+  };
+
+  updateScript = writers.writeBash "${pname}-update.sh" ''
+    set -euo pipefail
+
+    PATH=${
+      lib.makeBinPath [ curl common-updater-scripts nixfmt nix coreutils perl ]
+    }
+    perlexpr='if (/current version.{1,10}v(\d+)\.(\d+)\.(\d+)/i) { print "$1.$2.$3"; break; }'
+
+    oldVersion="$(nix-instantiate --eval -E "with import ./. {}; lib.getVersion ${pname}" | tr -d '"')"
+    latestVersion="$(curl -sS https://www.roomeqwizard.com/index.html | perl -ne "$perlexpr")"
+
+    if [ ! "$oldVersion" = "$latestVersion" ]; then
+      update-source-version ${pname} "$latestVersion" --version-key=version --print-changes
+      nixfmt "pkgs/applications/audio/roomeqwizard/default.nix"
+    else
+      echo "${pname} is already up-to-date"
+    fi
+  '';
+
+in runCommand "roomeqwizard-${version}" {
```

Comment:
Why is this no mkDerivation?

Reply:
I can change it to be one, but there's effectively no need. I was inspired by other pkgs using install4j to do it like this.

Reply:
Packages usually use mkDerivation and runCommand is more for simple additions to something. It makes it easier if all packages are similar.